website/docs/guides/oidc.mdx

```diff
@@ -0,0 +1,71 @@
+---
+title: Common OIDC provider configurations
+---
+
+This page provides guides for configuring Weave GitOps with the most common OIDC providers.
+
+## Google
+
+1. Obtain the client ID and secret by following the [official guide](https://developers.google.com/identity/openid-connect/openid-connect)
+   from Google.
+1. Configure Weave GitOps:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+       name: oidc-auth
+       namespace: WEAVE_GITOPS_NAMESPACE
+   stringData:
+       clientID: CLIENT_ID_FROM_STEP_1
+       clientSecret: CLIENT_SECRET_FROM_STEP_1
+       issuerURL: https://accounts.google.com
+       redirectURL: http://YOUR_WEAVE_GITOPS_DOMAIN/oauth2/callback
+       customScopes: openid,email
+   ```
+
+## Azure AD
+
+1. Obtain the client ID and secret by following the [official guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+   from Microsoft.
+1. Configure Weave GitOps:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+       name: oidc-auth
+       namespace: WEAVE_GITOPS_NAMESPACE
+   stringData:
+       clientID: CLIENT_ID_FROM_STEP_1
+       clientSecret: CLIENT_SECRET_FROM_STEP_1
+       issuerURL: https://login.microsoftonline.com/TENANT_ID/v2.0
+       redirectURL: http://YOUR_WEAVE_GITOPS_DOMAIN/oauth2/callback
+       customScopes: openid
+       claimUsername: sub
+   ```
```

Comment:
Maybe we want to talk about the groups on Azure?

And recommend that users use roles https://learn.microsoft.com/en-us/security/zero-trust/develop/configure-tokens-group-claims-app-roles ?

Reply:
👍🏻 I will need to set this up one more time to see what specifically is necessary for getting the groups.

Reply:
Tested this once more and added a step to configure group claims.

Reply:
Did this get pushed?

```yaml
       customScopes: openid
```

Reply:
The funny thing is you don't need to request any other scope if the groups claim is configured. The claim will be pushed nonetheless. 🤷🏻‍♂️ Microsoft doing Microsoft things.